gdi/monitors-hosts/filesystems.rst

```diff
@@ -4,10 +4,78 @@ Free disk space (filesystems)
 =============================
 
 .. meta::
-   :description: Use this Splunk Observability Cloud integrationfor the file systems / filesystems monitor. See benefits, install, configuration, and metrics
+   :description: Use this Splunk Observability Cloud integration for the file systems / filesystems monitor. See benefits, install, configuration, and metrics
 
 .. note:: If you are using the Splunk Distribution of OpenTelemetry Collector and want to collect file system utilization metrics, use the native OTel component :ref:`host-metrics-receiver`.
 
+The :ref:`Splunk Distribution of OpenTelemetry Collector <otel-intro>` uses the :ref:`Smart Agent receiver <smartagent-receiver>` with the ``filesystems`` monitor type to to retrieve file system metrics.
```

Comment:
There's 2 "to" at the end of the sentence!

Reply:
Thanks!